index.html

```diff
@@ -10,171 +10,214 @@
-        <div class="project-tag project-info">
-          <ul>
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
+      <div class="each-project-container">
+        <div class="work-collection project5">
+          <h3 class="project-title project-info">Profesional Art 5<br>Printing Data</h3>
+          <p class="project-description project-info">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <div class="project-tag project-info">
+            <ul>
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button type="button" class="project-button">See Project</button>
         </div>
-        <button type="button" class="project-button">See Project</button>
       </div>
-      <div class="work-collection project6">
-        <h3 class="project-title project-info">Profesional Art<br>Printing Data</h3>
-        <p class="project-description project-info">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-        <div class="project-tag project-info">
-          <ul>
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-          </ul>
+      <div class="each-project-container">
+        <div class="work-collection project6">
+          <h3 class="project-title project-info">Profesional Art 6<br>Printing Data</h3>
+          <p class="project-description project-info">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <div class="project-tag project-info">
+            <ul>
+              <li>css</li>
+              <li>html</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </div>
+          <button type="button" class="project-button">See Project</button>
         </div>
-        <button type="button" class="project-button">See Project</button>
       </div>
     </div>
   </section>
   <section class="about-me">
     <h2 class="about-me-title">About Me</h2>
     <p class="about-me-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
     <button type="button" class="button2">Get My Resume</button>
-    <div class="background-img2"></div>
-    <div class="vector-line"></div>
+    <!--<div class="background-img2"></div>-->
+    <div class="vector-line2"></div>
+  </section>
```

Comment:
- Again, you seem to have introduced a bug after the last review, as noted earlier, there is a missing background in this section, please fix this 🙏 . I am making this change mandatory for the same reason I stated above

Your project:
<img width="949" alt="evvjvjenvj" src="https://user-images.githubusercontent.com/73607512/206566658-e4788370-02de-4031-898f-a48a55cc339d.PNG">

Figma template:
<img width="546" alt="efwefef" src="https://user-images.githubusercontent.com/73607512/206566687-105d2205-84cc-45e8-b2c6-e78baa75c466.PNG">

Reply:
@mckent05 Thank you for your review. I deleted the commented code that you kindly told me. However, the images that you are telling me to put are there, were always in there. I asked my partner to review my portfolio in his computer to check if I have a bug but the images showed up in his computer as well so I don't why you could not be able to see them.